src/main/java/bio/terra/service/filedata/azure/blobstore/AzureBlobStorePdao.java

```diff
@@ -72,24 +72,15 @@ public FSFileInfo copyFile(
         profileDao.getBillingProfileById(fileLoadModel.getProfileId());
 
     BlobContainerClientFactory destinationClientFactory =
-        getTargetDataClientFactory(profileModel, storageAccountResource, false);
+        getTargetDataClientFactory(profileModel, storageAccountResource, ContainerType.DATA, false);
```

Comment:
Shouldn't  be `Destination` instead of `Target`? The term `Destination` is used elsewhere.

Reply:
Ah yeah, looks like this was mis-named.